### ISSUE TITLE
Suggestion/query

### ISSUE THREAD

**Issue by pixeltester** [cy33hc/switch-smb-client#1]
Thanks for the homebrew apps, I'm just wondering why you don't roll ftp, smb and any others you have planned into a single app with protocol selection? Especially as they share gui, layout style and libraries. Wouldn't that be not only more convenient for users as well as potentially simplifying development and maintenance?

**Comment by cy33hc** [comment]
@pixeltester That's what I will be doing next to merge various clients into one. I created them separately to test the waters and formula some commonality.

**Comment by pixeltester** [comment]
I see, thanks for the response. I was also wondering how the nro sizes ended up being so huge, for example 8MB to 28MB between the previous releases of switch-ftp-client!

**Comment by cy33hc** [comment]
@pixeltester That just a font file that includes languages that the switch doesn't natively have like Arabic, Thai etc...

**Comment by pixeltester** [comment]
Hey @cy33hc I hope you are well. I was just curious whether you still planned to merge clients into a single homebrew?

**Comment by cy33hc** [comment]
@pixeltester Yeah, I'm still planning on merging them. I done it for the PS4 and VITA. I haven't got to switch yet.

**Comment by cy33hc** [comment]
@pixeltester New project with merged FTP and SMB. Introduced WebDav client. https://github.com/cy33hc/switch-ezremote-client
